doc/missions.md

```diff
@@ -52,6 +52,9 @@ Perform full catchup to Public Network, but in parallel to improve performance.
 
 Perform full parallel catchup to Public Network, but with increased parallelism.
 
```

Comment:
Putting a comment here so we don't forget - getting-started.md needs to be updated to mention that `helm` needs  to be installed for the v2 parallel catchup mission.

Reply:
Updated the docs to mention `Helm`